cmd/shortener/main.go

```diff
@@ -18,22 +16,16 @@ func main() {
 	configs.Parse()
 
 	// инициализируем logger
-	logs := logger.NewLogger(logger.WithLevel(configs.flagLogLevel))
+	logs := logger.NewLogger(logger.WithLevel(configs.LogLevel))
 	logs.Info("Start logger")
 
 	// инициализируем хранилище
-	storage := mapstorage.NewMapURL()
-	logs.Info("Storage created")
+	repo := configs.repository
+	logs.Info("Connecting DB", repo)
+	defer repo.Close()
```

Comment:
Стоит вызывать `defer` сразу после иницализации - так не потеряем контекст, и в случае сбоя в ` logs.Info(...)` `defer`  все равно выполнится.
```suggestion
	repo := configs.repository
	defer repo.Close()
	logs.Info("Connecting DB", repo)
```